homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -68,20 +96,36 @@ async def async_setup_entry(hass, config_entry):
     certfile = hass.config.path(config_entry.data[CONF_CERTFILE])
     ca_certs = hass.config.path(config_entry.data[CONF_CA_CERTS])
 
-    bridge = Smartbridge.create_tls(
+    bridge = LIPSmartbridge.create_tls(
```

Comment:
Have you tested this on a non-pro bridge?

Reply:
I ordered a non-pro bridge. It should be here tomorrow or friday for testing.

Reply:
I'm pretty sure I've added sufficient guarding in `LIPSmartbridge` to handle an empty response, invalid response, or an exception coming back from the non-pro bridge.  I'll make it less forgiving if the actual device allows once I get it.

Reply:
The non-pro bridge showed up today and it works as expected.